lib/pure/times.nim

```diff
@@ -16,7 +16,7 @@
                       # of the standard library!
 
 import
-  strutils
+  strutils, pegs
```

Comment:
No pegs dependencies please. Compile-times are important. (Is it even used anywhere?)